lib/checks/navigation/heading-order-evaluate.js

```diff
@@ -1,19 +1,50 @@
-function headingOrderEvaluate(node, options, virtualNode) {
-	const ariaHeadingLevel = virtualNode.attr('aria-level');
-	const nodeName = virtualNode.props.nodeName;
+import cache from '../../core/base/cache';
+import { querySelectorAllFilter, DqElement } from '../../core/utils';
+import { isVisible } from '../../commons/dom';
+
+function getLevel(vNode) {
+	const ariaHeadingLevel = vNode.attr('aria-level');
+	const nodeName = vNode.props.nodeName;
 
 	if (ariaHeadingLevel !== null) {
-		this.data(parseInt(ariaHeadingLevel, 10));
-		return true;
+		return parseInt(ariaHeadingLevel, 10);
```

Comment:
Should probably test the validity of this, so that `aria-level="-1"` doesn't cause axe to treat it as an iframe 👿